src/components/Avatar/Avatar.jsx

```diff
@@ -47,9 +53,16 @@ const Avatar = ({
     return src ? {} : { backgroundColor: getElementColor(backgroundColor) };
   }, [src, backgroundColor, customBackgroundColor]);
   const sizeStyle = useMemo(() => {
-    return customSize ? { height: `${customSize}px`, width: `${customSize}px` } : {};
+    return customSize ? { height: customSize, width: customSize } : {};
   }, [customSize]);
 
+  ariaLabel = ariaLabel || (typeof tooltipProps?.content === "string" ? tooltipProps.content : undefined);
+  if (withoutTooltip) {
+    tooltipProps?.content && delete tooltipProps.content;
```

Comment:
I understand why u did it but it is a bit weird,  I think you can just do something like this:
```suggestion
    const overrideTooltipProps = useMemo(() => {
    if (withoutTooltip) return undefined;
    else if (tooltipProps) {
     return {content: ariaLabel, ...tooltipProps};
   }
   else {
     return {content: ariaLabel};
   }
    }, [ariaLabel, tooltipProps]);
```

Reply:
put it in use memo for to be more organized and because we creating a new object... but can also be outside of use memo